src/chi/components/steps/steps.scss

```diff
@@ -0,0 +1,84 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+
+  .a-steps {
+    display: table;
+    padding: 0;
+    table-layout: fixed;
+    width: 100%;
+
+    & > li {
+      color: $brand-color;
+      display: table-cell;
+      font-size: $text;
+      position: relative;
+      text-align: center;
+      z-index: 0;
+
+      &:before {
+        background-color: $text-white;
+        border-radius: 50%;
+        content: '\25CF';
+        display: block;
+        font-size: 1.85rem;
+        height: 1.125rem;
+        line-height: 1rem;
+        margin: 0 auto 0.25rem;
```

Comment:
After we alter the line-height on the labels, it's going to push the labels up. We can add that space back by increasing the bottom margin here to `0.50rem`